docs/modules/write-the-docs/pages/glossary.adoc

```diff
@@ -0,0 +1,653 @@
+
+-:: The corresponding phrase is "minus sign."
+It can be used in the UI to represent the "collapse" command for a list or collapsible element.
+
+== A
+
+a versus an:: Use the article "a" before words that begin with consonants.
++
+Use "an" before words that begin with vowels, an "ef" sound (as in "an FAQ"), or an unsounded "h" (as in "an hour").
+
+-able:: Do not add "-able" as a suffix when the result is not a real word.
+Consult a xref:write-the-docs:overview.adoc#ga-opennms-docs-dictionaries[dictionary] to see if the final construction is a valid use of the suffix.
++
+[options="header", cols="2,2"]
+|===
+| Incorrect
+| Correct
+
+| orderable
+| doable +
+scalable
+
+|===
+
+[[glossary-abort]] abort:: Avoid.
+"End" should be used for operations, "exit" for programs, "stop" for hardware operations, and "cancel" for requests or processes.
+
+[[glossary-above-below]] above, below:: Avoid when referring to information in text.
+Instead, use "preceding" and "following" (or "previous" and "next") if you cannot make specific cross-references.
+
+[[glossary-active-inactive]] active, inactive:: Do not use to describe options, functions, or items that users can select.
+Use <<glossary-available-unavailable, available and unavailable>> instead.
+
+affect, effect:: Use "affect" (verb) to mean "influence."
+Use "effect" (noun) to mean "result."
++
+.Use of "affect" and "effect" in sentences
+====
+Even minor changes affect the performance of the database.
+
+Even minor changes have an effect on the performance of the database.
+====
+
+[[glossary-all-always]] all, always:: Do not use.
+Avoid all-encompassing terms unless they are accurate descriptions.
+
+and/or:: Avoid using this construction.
+Choose either "and" or "or," or rewrite the sentence.
++
+If avoiding the construction makes a sentence long or cumbersome, however, it's okay to use.
++
+.Use of "or" and "and/or" in sentences
+====
+You can save the document under its current name or under a new name.
+
+Will the new version contain information on how to write object-oriented code and/or use the class libraries?
+====
+
+[[glossary-available-unavailable]] available, unavailable:: Use to describe options or functions that users can select and deselect.
+Do not use <<glossary-active-inactive, active, inactive>>, <<glossary-enabled-disabled, enabled, or disabled>>.
++
+See also <<glossary-dimmed, dimmed>>.
+
+== B
+
+back up, back out:: Verb; two words.
++
+.Use of "back up" and "back out" in sentences
+====
+Do you plan to back up your files?
+
+If something goes wrong, you can back out.
+====
+
+backup, backout:: Noun and adjective.
++
+.Use of "backup" and "backout" in sentences
+====
+Did you create a backup for that?
+
+We need a backout plan.
+====
+
+[[glossary-box]] box:: When referencing the UI, use "box" instead of <<glossary-field, field>> to refer to any box _except_ a check box, a <<glossary-list, list>>, or a menu box.
```

Comment:
```suggestion
[[glossary-box]] box:: When referencing the UI, use "box" instead of <<glossary-field, field>> to refer to any box _except_ a check box, a <<glossary-lists, list>>, or a menu box.
```